.github/workflows/dependency-update.yaml

```
@@ -13,29 +13,66 @@ jobs:
+
       - name: Extract Version
         id: extract_version
+        if: steps.check_commits.outputs.has_commits == '1'
         shell: nu {0}
         run: |
           let version = (open Cargo.toml | get package.version | str trim)
           $"version=($version)" | save --force $env.GITHUB_OUTPUT
 
       - name: Create GitHub Release
+        if: steps.check_commits.outputs.has_commits == '1'
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.11)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)